routes/api/gameApi.js

```diff
@@ -13,7 +13,6 @@ const sequelize = require("../../config/connection");
  */
 
 router.get("/", withAuth, async (req, res) => {
-  console.log(req.user);
   try {
     const userId = req.user.id;
```

Comment:
Can you please restructure the code?

`const { userId } = req.user;`

Reply:
but user don't have UserID, user have 'id'

Reply:
My bad! Awesome work! Sorry!

Reply:
That's alright